### ISSUE TITLE
Encoding errors

### ISSUE THREAD

**Issue by MariaCamilaCubides** [sebhildebrandt/systeminformation#646]
**Describe the bug**
Hi. I was doing some tests and I found some errors in my devices related to #616 issue. I found an error in `users` with the character ñ. And the other issue is with the accents (á, é, ó) in `graphics` and `networks.`

![image](https://user-images.githubusercontent.com/70594665/150614560-5d1e3b36-fb7f-42dc-9f7d-e684236db422.png)
![image](https://user-images.githubusercontent.com/70594665/150615782-d8b5a3a4-81f8-464a-b5be-e5a93f10284c.png)

I already had this kind of issue and it was solved in other devices sending commands through PowerShell and adding `$OutputEncoding = [System.Console]::OutputEncoding = [System.Console]::InputEncoding = [System.Text.Encoding]::UTF8 ; ` so I did some test and notice that this issue occurs in devices with Windows 8.1 and PowerShell 4.0

I tested the `$OutputEncoding` directly in PowerShell and then I run some commands whose results have special characters and it started failing.

**Comment by sebhildebrandt** [comment]
@MariaCamilaCubides ...  so I don't know If I understood you correctly. You mean, that I should just SKIP the `$OutputEncoding = [System.Console]::OutputEncoding = [System.Console]::InputEncoding = [System.Text.Encoding]::UTF8 ; ` ? 

Can you verify this on your side - when you blank out the `_psToUTF8` variable in the `lib/utils.js` ... Dows it then give correct output?

Thank you for your support.

**Comment by sebhildebrandt** [comment]
@MariaCamilaCubides I testet the current version once again (no changes) with a user with special characters in the name and it worked fine. Tests were done on Windows 11 ...

Can you maybe provide more detailed information about your environment?

**Comment by MariaCamilaCubides** [comment]
@sebhildebrandt I tried the current version on Windows 10, 11 and it works fine. I have this problem on Windows 8.1, however, I did a different test on Windows 8.1 and found that the problem is related to the PowerShell version.
The default version of PowerShell on Windows 8.1 is version 4.0, but this version doesn't support encoding which is set with `_psToUTF8`, I also removed this and tried sending the command alone, without encoding the information, but the error persists.
On the same device that has the problem, I updated the version of PowerShell to 5.1, and this way the error disappears.
